02-express-tutorial/app.js

```diff
@@ -1 +1,84 @@
-console.log('Express Tutorial')
+const express = require('express');
+const app = express();
+const { products } = require("./data")
+
+
+/**
+ * Create HTTP server
+ */
+// const http = require('http')
+const port = 3000;
+// app.set('port', port)
+// const server = http.createServer(app)
+// server.listen(port, () => {
+//     console.log(`app listening at http://localhost:${port}`)
+// })
+
+app.listen(port, () => {
```

Comment:
Typically the `listen()` call happens after you have configured all of your routes and middlwares.  i.e. this should go towards the end of the file.